src/components/map/MapService.js

```diff
@@ -370,6 +370,19 @@ goog.require('ga_urlutils_service');
           layers: params.LAYERS,
           format: 'image/png',
           service: 'WMS',
+          version: '1.1.1',
+          request: 'GetMap',
+          srs: 'EPSG:4326',
+          bbox: '{westProjected},{southProjected},' +
+                '{eastProjected},{northProjected}',
+          width: '256',
+          height: '256',
+          styles: 'default',
+          transparent: 'true'
+        } || {
+          layers: params.LAYERS,
+          format: 'image/png',
+          service: 'WMS',
           version: '1.3.0',
           request: 'GetMap',
           crs: 'CRS:84',
```

Comment:
Yes put the code I gave you. Because actually in your code the first object is always returs.

```
wmsParams = {props1} || {props2}

=>>> so wmsParams always equals to {props1}
```